cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -88,6 +88,14 @@ message CassandraConfigRole {
     optional TaskConfig cassandraYamlConfig = 11;
     // mesos role to be used to receive resource offers
     optional string mesosRole = 12;
+
+    /**
+     * A pre-defined data directory specifying where cassandra should write it's data.
+     * NOTE:
+     * This field will be removed once MESOS-1554 is released and the framework will
+     * be able to allocate the data volume itself.
+     */
+    optional string preDefinedDataDirectory = 13;
```

Comment:
A field can be removed from a protobuf later on, we just have to make sure that we don't re-use `13` as a fields cardinal value.  I'll change this to be something much higher and unlikely to ever be hit naturally.